radix-engine-tests/assets/serialized_receipts/basic_subintents/create-accounts.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Hmm I'm surprised this didn't error because they weren't regenerated... In a separate PR, 

I would suggest in a separate PR that we either:
* Move this into the scenario `dumper.rs` (probably slightly easier)
* Keep it separate, and use a `FolderContentAligner`, and have separate update/validate methods, and add its `update` function to `./update-scenarios.sh` so it will run when someone updates a scenario.

Reply:
Sounds good to me. Will move this into the scenario in a separate PR.